Cargo.toml

```diff
@@ -16,6 +16,7 @@ members = [
     "boards/rp-pico",
     "boards/solderparty-rp2040-stamp",
     "boards/sparkfun-pro-micro-rp2040",
+    "boards/arduino_nano_connect",
```

Comment:
This list was in alphabetical order - can you put this between adafruit-trinkey-qt2040 and pimoroni-pico-explorer?

Reply:
Can do